### PR TITLE
EVG-7840: Evergreen logo in navbar

### PR DESCRIPTION
**Pull request by tgrander**
Screen Shot 2020-05-11 at 2 46 13 PM

### CODE REVIEW

File: src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -1,39 +1,50 @@
 import React from "react";
 import styled from "@emotion/styled/macro";
-import { useAuthDispatchContext, useAuthStateContext } from "context/auth";
+import { useAuthStateContext } from "context/auth";
 import { Layout } from "antd";
+import { EvergreenLogo } from "components/icons";
+import { Subtitle } from "@leafygreen-ui/typography";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Link } from "react-router-dom";
+import { paths } from "constants/routes";
```
Comment (khelif96) [inline src/components/Navbar/index.tsx:9]: Eslint should be sorting these automatically. Theres a rule that External package imports should come before our imports. Guess its not working  ¯\\_(ツ)_/¯

File: src/components/icons/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export const EvergreenLogo = () => (
+  <svg
```
Comment (khelif96) [inline src/components/icons/index.tsx:4]: You should run this through [svgomg](https://jakearchibald.github.io/svgomg/). Get a 50% reduction in the svg size and remove allot of the unnecessary svg filler

Comment (tgrander) [reply src/components/icons/index.tsx]: good idea! totally forgot

File: cypress/integration/auth.ts [hunk 3]
```diff
@@ -1,34 +1,28 @@
 /// <reference types="Cypress" />
 /// <reference path="../support/index.d.ts" />
 
-describe("Auth", function () {
-  it("Unauthenticated user is redirected to login page after visiting a private route", function () {
+describe("Auth", function() {
```
Comment (khelif96) [inline cypress/integration/auth.ts:4]: => 😁

**Pull request merged**